File: Outlets/ViewController.m
//
//  ViewController.m
//  Outlets
//
//  Created by Daniela Mendez on 04/02/25.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *LabelOne;
@property (weak, nonatomic) IBOutlet UILabel *LabelTwo;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    //Escribimos lo que queremos que aparezca en el output
    [_LabelOne setText:@"Hola soy la linea 1"];
    [_LabelTwo setText:@"Soy linea 2"];
}


@end

File: Date Picker/ViewController.swift
//
//  ViewController.swift
//  Date Picker
//
//  Created by Daniela Mendez on 04/02/25.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var datePicker: UIDatePicker!
    
    @IBOutlet weak var dateLabel: UILabel!
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func readDate(_ sender: Any) {
        
        //Obtenemos la fecha seleccionada
        let selectedDate = datePicker.date
        
        //Creamos un formato para la fecha
        let dateFormatter = DateFormatter()
        dateFormatter.dateStyle = .long
        dateFormatter.timeStyle = .none
        
        //Convertimos la fecha en un string
        let dateString = dateFormatter.string(from: selectedDate)
        
        dateLabel.text = dateString
        
    }
    
}

